clients/hierarchy/data.go

```diff
@@ -0,0 +1,26 @@
+package hierarchy
+
+// Model represents the model returned by the heirarchy api
+type Model struct {
+	ID       string  `json:"id"`
+	Label    string  `json:"label"`
+	Children []Child `json:"children"`
+	Parent   Parent  `json:"parent"`
+}
+
+// Child represents a child item in the hierarchy model
+type Child struct {
+	ID               string `json:"id"`
+	Label            string `json:"label"`
+	URL              string `json:"url"`
+	NumberofChildren int    `json:"number_of_children"`
```

Comment:
I think there should be some omitempty's in these structs - number_of_children for one, possibly Parent and Children on the Model struct?